models/pendulum/pendulum.xml

```diff
@@ -1,15 +1,34 @@
 <mujoco model="pendulum">
-  <compiler angle="radian"/>
-  <statistic meansize="0.250205" extent="1.0008" center="0 0 -0.5"/>
+  <!-- Make some textures for the ground -->
+  <asset>
+    <texture name="grid" type="2d" builtin="checker" width="512" height="512" rgb1=".1 .2 .3" rgb2=".2 .3 .4"/>
+    <material name="grid" texture="grid" texrepeat="1 1" texuniform="true" reflectance=".2"/>
+  </asset>
+
   <worldbody>
-    <body name="base">
+    <!-- The floor -->
+    <geom name="floor" pos="0 0 0" size="0 0 .05" type="plane" material="grid" condim="3"/>
+
+    <!-- Some light -->
+    <light name="spotlight" diffuse=".8 .8 .8" specular="0.2 0.2 0.2" pos="0 -1 2" cutoff="1000"/>
+
+    <!-- The actual pendulum model -->
+    <body name="base" pos="0 0 0">
+      <!-- Dummy base inertia: for parser testing only -->
       <inertial pos="0 0 0" mass="2" diaginertia="1 1 1"/>
-      <body name="pendulum_link">
+      <body name="pendulum_link" pos="0 0 1.5">
         <inertial pos="0 0 -0.5" mass="1" diaginertia="0.083433 0.083433 0.0002"/>
-        <joint name="pendulum_joint" pos="0 0 0" axis="0 0 1" range="-3.1416 3.1416"/>
-        <geom name="pendulum_link_visual" size="0.02 0.5" pos="0 0 -0.5" type="cylinder" contype="0" conaffinity="0" group="1" density="0"/>
-        <geom name="pendulum_link_collision" size="0.02 0.5" pos="0 0 -0.5" type="cylinder"/>
+        <joint name="pendulum_joint" pos="0 0 0" axis="0 1 0" />
+        <geom name="pendulum_link_visual" size="0.02 0.5" pos="0 0 -0.5" 
+              type="cylinder" contype="0" conaffinity="0" group="1" density="0" 
+              rgba="1 0 0 1"/>
       </body>
     </body>
   </worldbody>
-</mujoco>
+
+  <!-- The actuator -->
+  <actuator>
+    <motor gear="10" joint="pendulum_joint"/>
```

Comment:
Why `gear="10"`?

Reply:
An arbitrary choice copy-pasted from somewhere else: just got rid of it.